xml/snmp_sysdescr.xml

```diff
@@ -1820,6 +1820,47 @@ Copyright (c) 1995-2005 by Cisco Systems
     <param pos="0" name="os.product" value="Wireless Controller"/>
   </fingerprint>
 
+  <fingerprint pattern="Meraki (MX[A-Z0-9]{2,10})">
```

Comment:
Was this intentionally unanchored? 👀 

I think the best practice over in the recog repo is to have explicit start+end anchors

```
  <fingerprint pattern="^Meraki (MX[A-Z0-9]{2,10})$">
```

Reply:
Raised a PR: https://github.com/rapid7/recog/pull/612